reviews/2022-k3s-annual.md

```diff
@@ -0,0 +1,208 @@
+* [Traefik](https://containo.us/traefik/) for ingress
+* [Klipper-lb](https://github.com/k3s-io/klipper-lb) as an embedded service loadbalancer provider
+* [Kube-router](https://www.kube-router.io/) for network policy
+* [Helm-controller](https://github.com/k3s-io/helm-controller) to allow for CRD-driven deployment of helm manifests
+* [Kine](https://github.com/k3s-io/kine) as a datastore shim that allows etcd to be replaced with other databases
+* [Local-path-provisioner](https://github.com/rancher/local-path-provisioner) for provisioning volumes using local storage
+* [Host utilities](https://github.com/k3s-io/k3s-root) such as iptables/nftables, ebtables, ethtool, & socat
+
+Additionally, K3s simplifies Kubernetes operations by maintaining functionality for:
+
+* Managing the TLS certificates of Kubernetes components
+* Managing the connection between worker and server nodes
+* Drop-in manifest apply for all built-in and custom resources
+* Managing an embedded etcd cluster
+
+### Accepted into CNCF Sandbox
+
+**K3s was accepted as a CNCF Sandbox project on August 19, 2020**
+
+- Sandbox proposal [pr #447](https://github.com/cncf/toc/pull/447)
+- Sandbox proposal [doc](https://github.com/cncf/toc/blob/main/proposals/sandbox/k3s.md)
+
+### Alignment with Cloud Native Computing Foundation
+
+K3s falls in the scope of [CNCF Runtime TAG](https://github.com/cncf/tag-runtime). We presented an overview of K3s to TAG Runtime, which can be viewed [here](https://youtu.be/ptoiC8fAPms).
+
+
+**Key features added**:
+
+- containerd v1.6.x
+- etcd 3.5
+- NATS JetStream added as supported storage backend
+- MicroOS support
+- Increased focus on automated testing across multiple OS/configs
+- encryption key secret rotation
+- bootstrap reconciliation refactor
+- dual-stack support
+- IPv6-only support
+- Automatic detection of nvidia container runtime
+- Automatic detection of systemd cgroup manager
+- Control-plane egress proxy (similar to Konnectivity service) using existing agent websocket tunnel
+
+## Annual Review Contents
+
+### Include a link to your project’s devstats page.
+Here is a link to our devstats page: https://k3s.devstats.cncf.io/d/8/dashboards?orgId=1&refresh=15m
+
+
+Github specific stats:
+
+| Statistic | Sandbox | 2021 | 2022 | 
+| -- | -- | -- | -- |
+| [GitHub Stars](https://github.com/k3s-io/k3s/stargazers) | ~14k | ~18k | ~21k | 
+| [Commits](https://github.com/k3s-io/k3s/commits/master) | ~75/month | ~65/month | [~100/month](https://github.com/k3s-io/k3s/pulse/monthly)  |
+| [Merged PRs](https://github.com/k3s-io/k3s/pulls) | ~35/month | ~45/month | [~66/month](https://github.com/k3s-io/k3s/pulse/monthly)  | 
+| [Releases](https://github.com/k3s-io/k3s/releases) | 100 | 171 | 189 |
+
+### How many maintainers do you have, and which organizations are they from?
+
+Our maintainers list: https://github.com/k3s-io/k3s/blob/master/MAINTAINERS
+
+Currently, the project has 17 maintainers, all with SUSE:
```

Comment:
To build a healthy and sustainable community, you need to have maintainers from more than one company.